modes/from-image/docker-compose.yml

```diff
@@ -113,17 +136,21 @@ services:
       COOKIE_SECURE_POLICY: ${API_SERVER_COOKIE_SECURE_POLICY}
       S3_AVATAR_FILES_BUCKET_ENDPOINT: ${API_SERVER_S3_AVATAR_FILES_BUCKET_ENDPOINT}
       S3_AVATAR_FILES_BUCKET_NAME: ${API_SERVER_S3_AVATAR_FILES_BUCKET_NAME}
-      S3_AVATAR_FILES_BUCKET_KEY_ID: ${MINIO_ACCESS_KEY}
-      S3_AVATAR_FILES_BUCKET_KEY_SECRET: ${MINIO_SECRET_KEY}
+      S3_AVATAR_FILES_BUCKET_KEY_ID: ${MINIO_ROOT_USER}
+      S3_AVATAR_FILES_BUCKET_KEY_SECRET: ${MINIO_ROOT_PASSWORD}
       S3_AVATAR_FILES_BUCKET_PUBLIC_URL_PREFIX: ${API_SERVER_S3_AVATAR_FILES_BUCKET_PUBLIC_URL_PREFIX}
 
   blockchain-indexer:
-    image: circlesubi/blockchain-indexer:dev
```

Comment:
is this going to remain like this? without  a docker image?or is just temporally until an image is build?